File: AWS-Interfaces/js-interface/006-security-group-delete.js
// Load the AWS SDK for Node.js
var AWS = require('aws-sdk');
// Set the region 
AWS.config.update({region: 'us-east-1'});

// Create EC2 service object
var ec2 = new AWS.EC2({apiVersion: '2016-11-15'});

// CHANGE
var params = {
   GroupId: 'sg-03b38c738017bafa5'
};

// Delete the security group
ec2.deleteSecurityGroup(params, function(err, data) {
   if (err) {
      console.log("Error", err);
   } else {
      console.log("Security Group Deleted");
   }
});

File: AWS-Interfaces/python-interface/006-security-group-delete.py
import boto3
from botocore.exceptions import ClientError

# Create EC2 client
ec2 = boto3.client('ec2', region_name='us-east-1')

# Delete security group
try:
    # CHANGE
    response = ec2.delete_security_group(GroupId='sg-04fe0d799cc9dd1e2')
    print('Security Group Deleted')
except ClientError as e:
    print(e)